projects/llm_load_test/toolbox/llm_load_test_run/templates/llm-load-test-config.yaml.j2

```diff
@@ -22,7 +22,7 @@ plugin_options:
 {% if llm_load_test_run_plugin == 'openai_plugin' %}
   host: "{{ llm_load_test_run_interface }}://{{ llm_load_test_run_host}}:{{ llm_load_test_run_port}}"
   endpoint: "{{ llm_load_test_run_endpoint }}"
-  model_name: "/mnt/models/"
+  model_name: "{{ llm_load_test_run_model_id }}"
```

Comment:
I think you can move this up out of the if statement

Reply:
good catch, fixed